File: 30DaysOfCode/day03_introToConditionalStatements/introToConditionalStatements.rb
#!/bin/ruby

N = gets.strip.to_i

if (N % 2 == 1)
    puts "Weird"
else
    if (N >= 2 and N <= 5)
        puts "Not Weird"
    elsif (N >= 5 and N <= 20)
        puts "Weird"
    else
        puts "Not Weird"
    end
end

File: 30DaysOfCode/day03_introToConditionalStatements/introToConditionalStatements.pl
#!/usr/bin/perl

$N = <STDIN>;
chomp $N;

if ($N % 2 == 1) {
    print "Weird";
}
else {
    if ($N >= 2 && $N <= 5) {
        print "Not Weird";
    }
    elsif ($N >= 6 && $N <= 20) {
        print "Weird";
    }
    else {
        print "Not Weird";
    }
}
